_docs/tasks/security/https-overlay.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Mutual TLS over https services
+overview: This task shows how to enable mTLS on https services.
+
+order: 80
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This task shows how Istio Mutual TLS works with https services. It includes: 1)
+Deploy a https service without Istio sidecar; 2) Deploy a https service with
+Istio with mTLS disabled; 3) Deploy a https service with mTLS enabled. For each
+deployment, connect to this service and verify it works.
```

Comment:
this is a bit confusing to me

for 3) don't you need to use tcp proxying ? and for 2) it also means tcp proxying and metrics

Reply:
Yes, it's true and explained below.